### PR TITLE
docs(README): Add troubleshooting section to README

### PR DESCRIPTION
**Pull request by rainerschoe**
Thanks for this project! I think it is awesome!! With this I finally got Resolve running on my Arch-Linux installation after many years :-)

I ran into some audio issues related to my Arch installation using Pulseaudio.
I added a section to the `README.md` which might help others.

Feel free to merge or propose changes.
In an ideal world the installer could detect or ask for the installed sound system and then create the `distrobox` image accordingly. But I did not have time to look into it this far. So I thought adding the info to the `README.md` is better than nothing.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -214,6 +214,21 @@ podman container stop davincibox
...
 ```
 
+## Troubleshooting
+### No Audio output
+DaVinci Resolve uses ALSA. In `davincibox` this is supported via the `pipewire-alsa` plugin, which re-directs sound to pipewire.
+This however requires that the host system provides a pipewire server. If your host-system uses some other sound server, this might not work.
+
+You can either convert your host system to provide `pipewire` or change your `davincibox` instance to use any sound server you have running.
+
+For example, if your host-system uses `pulse-audio`, you can change `davincibox` as follows:
```
Comment (zelikos) [inline README.md:224]: Should be `pulseaudio`, no hyphen

File: README.md [hunk 2]
```diff
@@ -214,6 +214,21 @@ podman container stop davincibox
...
+
+```
+> distrobox-enter -n davincibox -- bash
```
Comment (zelikos) [inline README.md:227]: `distrobox-enter -n davincibox` on its own should invoke bash already, there shouldn't be a need to do so manually here

**Review by zelikos — CHANGES_REQUESTED** [review]
A couple of nitpicks, but otherwise LGTM

File: README.md (hunk 1 above)
Comment (rainerschoe) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (rainerschoe) [reply README.md]: done

Comment (rainerschoe) [reply README.md]: it actually gives me fish :-) nice :-) didn't think it is installed in the image

Comment (zelikos) [reply README.md]: That's strange. Fish isn't included, so it must be something else on your setup that's causing that.

**Review by zelikos — APPROVED** [review]

**Pull request merged**